autogpt_platform/backend/backend/server/v2/library/db.py

```diff
@@ -238,7 +240,8 @@ async def create_library_agent(
                         "graphVersionId": {"id": graph.id, "version": graph.version}
                     }
                 },
-            )
+            ),
+            include={"AgentGraph": True},
```

Comment:
I'd advise changing the return type of this function to `library_model.LibraryAgent` instead of the prisma model